scripts/common

```diff
@@ -86,6 +86,18 @@ else
   )
 fi
 
+if [[ -z "${RUST_HOST_TARGET:-default}" ]]; then
+  if [[ "${OSTYPE}" == "darwin"*  ]]; then
+    build_target="x86_64-apple-macosx10.7.0"
+  else
+    build_target="x86_64-unknown-linux-gnu"
+  fi
+else
+  build_target="${RUST_HOST_TARGET:-default}"
```

Comment:
The `:-` seems superfluous in this `else` arm (or is this required by `shellcheck`?)